### PR TITLE
Added _random module

### LINKED ISSUE #37: Add random number generating functions
Add a new module called `_random`. It should work like `_hash` and `_env` do (they get added in as boilerplate code on lines *514*-*662* in `ask.py`). Behind the scenes, this should use Pythons own built-in `random` library.

**It should have the following functions:**
```python
# Single items
_random.int(0, 8) # Returns a random integer between 0 and 8.
_random.float(1.5, 3.8) # Returns a random float between 1.5 and 3.8.
_random.item(['a', 'b', 'c']) # Returns a random element from the passed in list.

# Multiple items
_random.ints(0, 8, 3) # Returns 3 random integers between 0 and 8 in a list.
_random.floats(1.5, 3.8, 8) # Returns 8 random floats between 1.5 and 3.8.
_random.items(['a', 'b', 'c'], 4) # Returns 4 random elements from the passed in list.

# You can add more functions if you want but at least these ones as a start...
```

**Helpful links**
- https://docs.python.org/3/library/random.html
- https://www.w3schools.com/python/module_random.asp

### PR DESCRIPTION
**Pull request by just-code-it-anyway**
I have created the _random module, please look into it and please notify me if any changes required.

Closes #37

### CODE REVIEW

File: _random.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        parameters: 
+        start - start of the range
+        end - end of the range (inclusive)
+        count(optional) - number of random integers to return, default value is 1
+        Note: count <= end-start
+    """
+
+    if end-start < count:
+        raise ValueError("Count of the integers should not be greater than the input range !")
+    if count>1:
```
Comment (Buscedv) [inline _random.py:15]: Put spaces around operators (<, >, -, +, *, %, etc...). Not only on this line, but there are also a few other places.

File: _random.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import random
+
+def int(start, end, count=1):
+    """ 
+    A function to return 'n' random integers between a given range 'Start' to 'End'
```
Comment (Buscedv) [inline _random.py:5]: These methods are so simple that we don't need documentation comments, you can remove them.

**Review by Buscedv — CHANGES_REQUESTED** [review]
Nice work and thank you for your contribution 👍 !

I would like this to not be a separate file. Can you move the code into the `ask.py` file and have it automatically inserted into the transpiled code, check lines *514*-*662* in `ask.py` and make this work in the same way that e.g. `Hash/_hash` and `Env/_env` work.

- You can put these function (inside a class called `Random`) under the `Hash` class.
- Then create an instance of the new `Random` class, called:  `_random` on line 647 (the same way as `_hash`.

Thank you again.

**Comment by just-code-it-anyway** [comment]
Thankyou for your support, i'll begin with the requested changes.

**Comment by just-code-it-anyway** [comment]
Hey @Buscedv , i have made the requested changes. Please have a look on the latest commit.

**Review by Buscedv — APPROVED** [review]
Approved! ✅

I have some refactorings that I'll add after merging this you can check them out if you want. They basically include some coding convention improvements and a few feature improvements, and some more functionality.

Awesome work! ⚡️👍

**Comment by just-code-it-anyway** [comment]
Thank you for your support on my carrier's first ever open source
contribution. This really helped me boost my confidence.😊

**Pull request merged**